specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-02-21-preview/Subscription.json

```diff
@@ -0,0 +1,214 @@
+{  
+   "swagger":"2.0",
+   "info":{  
+      "version":"1",
+      "title":"SubscriptionCreationClient",
+      "description":"Subscription creation client provides an interface to create, modify and retrieve azure subscriptions programmatically."
```

Comment:
"Subscription management client provides an interface to create and manage Azure subscriptions programmatically."